keras_nlp/tokenizers/__init__.py

```diff
@@ -12,6 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from keras_nlp.tokenizers.byte_pair_tokenizer import BytePairTokenizer
```

Comment:
Update the coauthor line in whichever commit that was to this...

`Co-authored-by: jessechancy <jessec1@andrew.cmu.edu>`

That's the email he is registered with on github. Also I think that's why the CLA check is failing.